src/tests/functional/plugin/shared/include/snippets/exp.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shared_test_classes/base/snippets_test_utils.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+typedef std::tuple<
+        InputShape,                  // Input 0 Shape
+        ov::element::Type,           // Element type
+        size_t,                      // Expected num nodes
+        size_t,                      // Expected num subgraphs
+        std::string                  // Target Device
+> ExpParams;
+
+class Exp : public testing::WithParamInterface<ov::test::snippets::ExpParams>,
```

Comment:
I remember you problem with `exp`. But do we really need the separate snippets test for `exp` or we can reuse eltwise tests where Snippets tokenize Subgraphs as well?

Reply:
This is a standard functional test, derived from `SnippetsTestsCommon`, the model is tokenized in a usual way.
I don't get your point, please elaborate.

Reply:
I meant that we already have plugin activation tests that [validates](https://github.com/openvinotoolkit/openvino/blob/149381ef8cda82b42d5dfaa0a4ab81700e0a7322/src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp#L26) `exp` as well. The model with single `exp` node is tokenized by Snippets and is tests as ordinary.
There is no difference between existing test and your new and we will have two the same tests